internal/cmd/root.go

```diff
@@ -47,9 +49,11 @@ func Root() *cobra.Command {
 	pflags := cmd.PersistentFlags()
 
 	pflags.BoolVar(&fAllowUnknown, "allow-unknown", true, "Display snippets without known executor")
+	pflags.BoolVar(&fIgnoreNameless, "ignore-nameless", true, "Ignore code blocks without name defined")
```

Comment:
```suggestion
	pflags.BoolVar(&fIgnoreNameless, "ignore-nameless", false, "Ignore code blocks without name defined")
```

Should probably be false by default.